contact.html

```diff
@@ -2,10 +2,70 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="author" content="Juan Nunez" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Contact</title>
+    <!--Font Links Section-->
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.gstatic.com"> 
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+    <!--CSS Links Section-->
+    <link rel="stylesheet" href="index.css">
+    <title>Contact Information</title>
 </head>
 <body>
+    <header>
+        <nav class="topnav">
+            <ul class="navul">             
+                <li class="navli"><a href="index.html">Home</a></li>|
+                <li class="navli"><a href="gallery.html">Gallery</a><li>|
+                <li class="navli"><a href="contact.html">Contact me</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <center>
+        <h1>Contact me</h1>
+        <p>Please use the form below to get in contact with me.</p>
+        <form action="#">
+        <label for="name">Name:</label>
+        <input type="name" name="name" id="name" placeholder="First and last name">
```

Comment:
Type "name" is invalid here; "name" isn't an HTML input type.  Consider replacing with a valid input type.